dat/spob/velyn.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<spob name="Velyn">
+ <lua>default.lua</lua>
+ <pos x="6766.763876" y="-1198.049998"/>
+ <GFX>
+  <space>003.webp</space>
+  <exterior>traderoom.webp</exterior>
+ </GFX>
+ <presence>
+  <faction>Proteron</faction>
+  <base>40.000000</base>
+  <bonus>20.000000</bonus>
+  <range>0</range>
+ </presence>
+ <general>
+  <class>M</class>
+  <population>10000</population>
+  <hide>1.000000</hide>
+  <services>
+   <land/>
+   <refuel/>
+   <commodity/>
+  </services>
+  <commodities/>
+  <description>The comptroller for the Proteron economy, Velyn is primarily a supercomputer with the prediction models for various aspects of the centralised Proteron economy. The place seems very busy, as you would expect for so vital a station, but it's rather small as a whole.</description>
```

Comment:
```suggestion
  <description>The comptroller for the Proteron economy, Velyn is primarily a supercomputer with the prediction models for various aspects of the centralized Proteron economy. The place seems very busy, as you would expect for so vital a station, but it's rather small as a whole.</description>
```

Reply:
Is American English expected for descriptions? If so, I apologise. I assumed the British standard was acceptable.

Reply:
Naev uses Oxford English, which I believe does ize instead of ise. But I suppose I'm not positive on that, so it's up to bobbens.